OrganizationChart.md

```diff
@@ -0,0 +1,196 @@
+        if (string.find(page.name,mypage) and mypage ~= page.name)
+        then
+              table.insert(crumbsChildren, {name = page.ref})
+        end
+    end
+    return crumbsChildren
+end
+
+function getFrontMatter(page)
+    return index.extractFrontmatter(space.readPage(page)).frontmatter
+end 
+
+function organizationChart(path)
+  path = path or editor.getCurrentPage()
+  local uml = {}
+  table.insert(uml, "```plantuml")
+  table.insert(uml, "@startuml")
+  table.insert(uml, "scale 1100 width")
+  --table.insert(uml, "scale 768 height") 
+  table.insert(uml, "<style>")
+  table.insert(uml, "root {")
+  table.insert(uml, "  HyperlinkColor #00000")
+  table.insert(uml, "}")
+  table.insert(uml, "</style>")
+  table.insert(uml, "!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Context.puml") 
+  table.insert(uml, "LAYOUT_TOP_DOWN()") 
+  table.insert(uml, "") 
+  local liste= children(path)
+  for i,pag in ipairs(liste) do
+        local frontM= getFrontMatter(pag.name)
+        table.insert(uml, "'"..pag.name) 
+        if frontM ~= nil and frontM.person ~= nil then
+          if frontM.person.inherit ~= nil then
+            local frontMParent= getFrontMatter(frontM.person.inherit)
+            frontM=mergeTablesRecursive(frontMParent,frontM)
+            --table.insert(uml, prettyPrint(frontM))
+          end
+          local person = frontM.person
+          --table.insert(uml, "'"..frontM.person.first_name) 
+          table.insert(uml, generatePerson(uml, person,path))
+        end
+  end
+  table.insert(uml, "")
+  table.insert(uml, "@enduml")
+  table.insert(uml, "```")
+  table.insert(uml, "")
+  local result= table.concat(uml, "\n")  
+  print(result)
+  return result
+  --system.invokeFunction("plantuml", result)
+end
+
+slashcommand.define {
+  name = "organizationchart",
+  run = function()
+    editor.insertAtCursor(organizationChart(), false, true)
+  end
+}
+```
+
+```space-lua
+-- add image
+person={}
+
+function person.insertImageFromFrontmatter()  
+  -- Get current page frontmatter  
+  local pageText = editor.getText()  
+  local fm = index.extractFrontmatter(pageText)  
+  local result="No image found" 
+    
+  -- Check if person.image exists in frontmatter  
+  local types={"person","company"}
+  for row,v in ipairs(types) do
+    if fm.frontmatter and fm.frontmatter[v] and fm.frontmatter[v].image then  
+      local imagePath = fm.frontmatter[v].image  
+      local name=""
+      if types == "company" then
+        name =(fm.frontmatter[v].name or "")
+      else
+        name=(fm.frontmatter[v].first_name or "") .. " " ..  (fm.frontmatter[v].last_name or "")
+      end
+        
+      -- Create markdown image syntax  
+      if imagePath ~= nil then 
+        result =  "![" .. name .. "|x200](" .. imagePath .. ")"  
+        break 
+      end
+    end  
+  end
+  return result
+end
+```
+
+
+\${organizationChart("work/ampere-regienov/contact") }
```

Comment:
This looks like a local test? Perhaps it's possible to include an example in this page as documentation?